tests/Integration/Behaviour/Features/Context/Domain/OrderPaymentFeatureContext.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+namespace Tests\Integration\Behaviour\Features\Context\Domain;
+
+use Behat\Gherkin\Node\TableNode;
+use PrestaShop\PrestaShop\Core\Domain\Order\Payment\Command\AddPaymentCommand;
+use PrestaShop\PrestaShop\Core\Domain\Order\Query\GetOrderForViewing;
+use PrestaShop\PrestaShop\Core\Domain\Order\QueryResult\OrderForViewing;
+use PrestaShop\PrestaShop\Core\Domain\Order\QueryResult\OrderPaymentForViewing;
+use PrestaShop\PrestaShop\Core\Domain\Order\QueryResult\OrderPaymentsForViewing;
+use PrestaShopException;
+use RuntimeException;
+use Symfony\Component\DependencyInjection\Exception\ServiceCircularReferenceException;
+use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
+
+class OrderPaymentFeatureContext extends AbstractDomainFeatureContext
+{
+    /**
+     * @When I add payment to order with id :orderId with the following properties:
```

Comment:
```suggestion
     * @When I add payment to order with id :orderId with the following details:
```
Following the trend "try to make it as human readable as possible" could we try to make this operations more like "And I pay order X using a check whose amount is 5.548"

I suggest it for this step because this step is actually a very human behavior: paying for an online order is something that happens in a classical ecommerce checkout process 😄